src/chrome/content/rules/Servage.net.xml

```diff
@@ -1,50 +1,37 @@
 <!--
 	For other Levonline coverage, see Levonline.com.xml.
 
+	Invalid certificate:
+		servage.com
+		www.servage.com
+		www.webmail.servage.net
 
-	Problematic domains:
-
-		- (www.)servage.com	(shows another domain; mismatched, CN: *.levonline.com)
-		- webmail.servage.net	(shows www, valid cert)
-
-
-	Partially covered domains:
-
-		- (www.)servage.com	(→ www.servage.net)
-		- webmail.servage.net	(→ secure)
-
-
-	Fully covered servage.net subdomains:
-
-		- (www.)	(^ → www)
-		- livesupport01
-		- secure
-		- static
+	Login required:
+		webmail.servage.com
 
 -->
 <ruleset name="Servage.net">
 
-	<target host="servage.*" />
+	<target host="servage.com" />
 	<target host="www.servage.com" />
-	<target host="*.servage.net" />
-
-
-	<securecookie host="^(?:livesupport01|secure|www)\.servage\.net$" name=".+" />
+	<target host="vps.servage.com" />
 
+	<target host="www.servage.net" />
+	<target host="secure.servage.net" />
+	<target host="static.servage.net" />
+		<test url="http://static.servage.net/img/favicon.ico" />
+	<target host="webmail.servage.net" />
+	<target host="www.webmail.servage.net" />
```

Comment:
```
cp.servage.net
images.servage.net
livesupport01.servage.net
```

There are (obsolete?) customer domains on `\{8}.c\d+.servage.net` and `vserver\d+.servage.com`

Reply:
ping @Bisaloo